pkg/query-service/app/logs/parser_test.go

```diff
@@ -102,8 +102,10 @@ var correctQueriesTest = []struct {
 func TestParseLogQueryCorrect(t *testing.T) {
 	for _, test := range correctQueriesTest {
 		Convey(test.Name, t, func() {
-			query, _ := parseLogQuery(test.InputQuery)
-
+			query, err := parseLogQuery(test.InputQuery)
+			if err != nil {
```

Comment:
Add something like So(err, ShouldBeNil)

Reply:
will `So(err, ShouldBeNil)`  print the error message ?
If No, we shouldn't use it here.

Reply:
Yes, the `so` function will show the value of the error message. You can try out by creating test cases with errors